src/libstd/ffi/os_str.rs

```diff
@@ -188,6 +188,16 @@ impl OsString {
     /// in the given `OsString`.
     ///
     /// The collection may reserve more space to avoid frequent reallocations.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
```

Comment:
Latest force push removes them.

> I removed the annotations in a lot of places, so please don't add them back.

Sure, but just because you removed the annotations doesn't mean we should enforce this everywhere. Personally, I prefer to leave in the `rust` annotation.